modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -79,6 +79,13 @@ endif::rosa-hcp[]
 * Service CIDR: 172.30.0.0/16
 * Pod CIDR: 10.128.0.0/16
 * Host prefix: /23
++
+ifdef::rosa-hcp[]
+[NOTE]
+====
+When using {hcp-title}, the static IP address `172.20.0.1` is reserved for the internal Kube API address. The Machine, Pod, and Service CIDRs ranges must not conflict with this IP.
```

Comment:
Spell out kubernetes, plus I don't think the machine/pod/service needs to be capitalized. Add "address" after the second IP

```suggestion
When using {hcp-title}, the static IP address `172.20.0.1` is reserved for the internal Kubernetes API address. The machine, pod, and service CIDRs ranges must not conflict with this IP address.
```